cloud-controller-manager/do/health_test.go

```diff
@@ -17,67 +17,74 @@ limitations under the License.
 package do
 
 import (
+	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
+	"strings"
 	"testing"
 
 	"github.com/digitalocean/godo"
 )
 
-func TestGodoHealthChecker_Name(t *testing.T) {
-	c := godoHealthChecker{}
-	if want, got := "godo", c.Name(); want != got {
-		t.Errorf("incorrect name\nwant: %#v \n got: %#v", want, got)
+func TestGodoHealthChecker(t *testing.T) {
```

Comment:
Note to reviewers: the pre-existing two tests were merged into a single test with two test cases.